global_body_planner/include/global_body_planner/planning_utils.h

```diff
@@ -271,6 +391,8 @@ bool isValidYawRate(const State &s, const Action &a, double t, int phase,
                     const PlannerConfig &planner_config);
 bool isValidState(const State &s, const PlannerConfig &planner_config,
                   int phase);
+bool isValidState(const State &s, const PlannerConfig &planner_config,
+                  int phase, double &max_height);
```

Comment:
double doesnt need ref &

Reply:
This one does, its an output (not const).

Reply:
Got it